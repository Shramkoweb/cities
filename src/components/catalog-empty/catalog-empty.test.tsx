import * as React from "react";
import {configure, shallow} from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";

import CatalogEmpty from "./catalog-empty";

configure({adapter: new Adapter()});

it(`Empty main page renders correctly`, () => {
  const EmptyCatalog = shallow(
    <CatalogEmpty currentCity={`Kiev`}/>
  );

  expect(EmptyCatalog).toMatchSnapshot();
});
