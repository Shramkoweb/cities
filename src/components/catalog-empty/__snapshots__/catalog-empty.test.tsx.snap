// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty main page renders correctly 1`] = `ShallowWrapper {}`;
